node/full_node.go

```diff
@@ -0,0 +1,349 @@
+package node
```

Comment:
I would love to see continuous git history for this code. Ideally `full_node.go` should be "detected" as moved & edited version of `node.go`. Probably, `node.go` you'll need two commits - first with move (node.go->full_node.go), second with creation of new node.go file.

Reply:
is there some way i can restore it?

Reply:
![73ggiq](https://user-images.githubusercontent.com/1138893/210381938-ac739994-a716-4a47-a001-9d2394bd284f.gif)